src/System.Data.SqlClient/tests/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+
+  <ItemGroup>
+    <ProjectReference Include="**\*.csproj" />

```

Comment:
This doesn't seem to include the test projects for CI build and running tests.

Reply:
What do you mean?